lib/util/getAutorestConfigInternal.ts

```diff
@@ -46,7 +46,7 @@ const getAutorest = async (version: string | undefined, rfs?: IFileSystem, readm
 };
 
 const getAutorestCoreVersion = async (_args?: any) => {
-  return "latest-installed";
+  return "2.0.4421";
```

Comment:
what's this for?

Reply:
it can't run in autorest 3.x, so we need declare 2.x version

Reply:
2.x has been deprecated. And why do we need autorest/core here?

Reply:
using autorest core to resolve readme file. I have fixed to use 3.x version.